code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/OrgEventFragment.java

```diff
@@ -30,7 +30,7 @@
  */
 public class OrgEventFragment extends Fragment {
 
-    private ImageView qrCodeImage;
+    private ImageView eventPoster;
```

Comment:
Wait, so are we no longer showing the event check in qr code?

Reply:
no, since we have the share function

Reply:
True, but currently the "Promote" button only gives us the promotional QR code. We also need to be able to share the event check in QR code.

Reply:
I thought that the ´Sharé’ button shared the check in qr

Reply:
Oh wait, you are so right. my bad